### DISCUSSION TITLE
TuringCourses/math_phys/math_analysis1/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#138] [General]
# TuringCourses/math_phys/math_analysis1/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/math_phys/math_analysis1/

**Comment by lhxcs** [comment]
对于课本内容掌握较为扎实，并且对习题有需求的同学，推荐一本教材，也是22级陈锦辉班助教推荐的书籍： 裴礼文的《数学分析中的典型问题与方法》。这本书以题目为主，基本都是数分中的经典例题，并且作者在例题中传达出分析的一些基本思想方法，做过之后对基本理论有了更深的理解，证明思路也清晰了许多，很有收获。但是书中有部分题（带有※记号）难度较大，超出课程的要求，可以不必深究，毕竟这本书根本上是面向数院数分的。

再强调一下，只建议能够跟上上课进度，对课本内容掌握较扎实，学有余力的同学额外做习题册进行补充，毕竟基础不牢地动山摇，而且熟练掌握课本对于课程要求来说已经足够。对于做这本习题的同学，如果掌握了书中标☆的题，那么在本课程中取得优异成绩基本是没有问题的。除了功利的应试角度之外，这本书对分析功底的提高也是大有裨益。

**Comment by leeJiawen** [comment]
Real Analysis: A Long-form mathematics textbook by Jay Cummings 也是一本很好的书！语言非常幽默轻松（甚至还有梗图）同时把很多东西讲得很清楚 配有很多visualization。尤其是在证明前会用浅显的语言讲一下证明的big picture，对初学非常友好！强烈推荐！
